InputAnswer/InputAnswer/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace InputAnswer
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int excitement = 0;
+             
+
+            Console.WriteLine("Hi! We are starting out on a great journey");
+            Console.WriteLine("You and I");
+            Console.WriteLine("Are you excited?");
+            Console.WriteLine("Type 1 for yes, 2 for no, and 3 for eh: ");
+
+            excitement = Convert.ToInt32(Console.ReadLine());
```

Comment:
Fix: A couple of things:
1) While it's shorter to express this all at one line, if there's an issue, it could be both harder to read and harder to diagnose since there are 2 things happening in this line - Reading from the console and Converting the response into an int. 

2) If I type "four" into the console, what happens when Convert.ToInt32() is called on that? What happens if I just hit enter? What happens if I enter a number larger than a 32-bit int? What happens if I enter a negative number?

3) If there is an exception, what happens? What do you want to happen?